lib/index-basic.js

```diff
@@ -0,0 +1,19 @@
+/**
+* Copyright 2012-2016, Plotly, Inc.
+* All rights reserved.
+*
+* This source code is licensed under the MIT license found in the
+* LICENSE file in the root directory of this source tree.
+*/
+
+'use strict';
+
+
+var Core = require('./core');
```

Comment:
Maybe worth an inline comment to note that [`Core` pulls in scatter](https://github.com/plotly/plotly.js/blob/master/src/plotly.js#L77). If I were looking to bundle my own, I'd start by copy/pasting this file, and I had to track down the inclusion of scatter.